spec/spec_helper.rb

```diff
@@ -18,7 +18,3 @@ def input(plugin, &block)
   end
 
 end
-
-RSpec.configure do |c|
-  c.include IrcHelpers
```

Comment:
Interesting this makes the test to crash, I really was expecting it to be harmless.  Does this means we are in trouble of using multiple ```RSpec.configure``` ? kinda makes sense, but still interesting.